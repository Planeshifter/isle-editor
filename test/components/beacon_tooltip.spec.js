// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import BeaconTooltip from 'components/beacon-tooltip';


// TESTS //

it( 'renders a beacon tooltip', () => {
	const tooltip = renderer.create(
		<BeaconTooltip title="Tooltip title" content="Tooltip content" />
	).toJSON();
	expect( tooltip ).toMatchSnapshot();
});
