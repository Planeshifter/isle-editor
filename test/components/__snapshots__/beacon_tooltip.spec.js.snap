// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a beacon tooltip 1`] = `null`;
